Mathlib/Data/Set/Card.lean

```diff
@@ -721,6 +721,13 @@ theorem ncard_congr {t : Set β} (f : ∀ a ∈ s, β) (h₁ : ∀ a ha, f a ha
   simp_rw [← Nat.card_coe_set_eq]
   exact Nat.card_congr (Equiv.ofBijective f' hbij)
 
+theorem ncard_coe {α : Type*} (s : Set α) :
+    s.ncard = Set.ncard (Set.univ : Set (Set.Elem s)) := by
+  apply Set.ncard_congr (fun a ha ↦ ⟨a, ha⟩)
+  · exact fun a ha ↦ by simp only [Set.mem_univ]
+  · simp [Subtype.mk_eq_mk]
+  · exact fun ⟨a, ha⟩ _ ↦ ⟨a, ha, rfl⟩
```

Comment:
Is this lemma used anywhere in the PR? If not, you could move it to a separate PR.

Reply:
You're right. I deleted them.